sentry-options-cli/src/main.rs

```diff
@@ -44,23 +44,56 @@ pub enum AppError {
     Schema(#[from] sentry_options_validation::ValidationError),
 }
 
-/// Required CLI arguments
+/// defines the CLI for sentry-options validation and processing
 #[derive(Parser, Debug)]
-#[command(name = "sentry-options")]
+#[command(name = "sentry-options-cli")]
 #[command(version, about, long_about = None)]
-struct Args {
-    #[arg(long, required = true, help = "root directory of the sentry options")]
-    root: String,
-
-    #[arg(long, required = true, help = "output directory for final json files")]
-    out: String,
-
-    #[arg(
-        long,
-        required = true,
-        help = "directory containing namespace schema definitions"
-    )]
-    schemas: String,
+struct Cli {
+    #[command(subcommand)]
+    command: Commands,
+}
+
+/// Available subcommands
+#[derive(Subcommand, Debug)]
+enum Commands {
```

Comment:
taking another look at the clap docs, I'd recommend using a builder instead: https://docs.rs/clap/latest/clap/builder/struct.Command.html